src/zimscraperlib/zim/creator.py

```diff
@@ -146,7 +150,35 @@ def config_indexing(
         self.__indexing_configured = True
         return self
 
+    def _is_illustration_metadata_name(self, name: str) -> bool:
+        """Return True if name is a valid illustration metadata name"""
+        return name.startswith("Illustration_")
+
+    def _get_illustration_metadata_details(
```

Comment:
this function is not very appealing. You want to know the image format and its dimensions. Why not returning this instead of building a string in some util function?
As per error handling, you have the choice to handle it when calling or inside the function but anyway, better safe than sorry and handle every exception. Currently if PIL cannot read (and it's not `UnidentifiedImageError` or if size could not be set to a tuple, it will crash

Reply:
Okay, now it won't crash quite as easily. Agreed that this could return a richer data-structure. Doing so would complicate the caller. Avoiding the complication in the caller by pushing the logging into this function smears the knowledge of the logging format in use around. It also prevents converting this whole thing from N logging calls to 1 call contains N items; the latter may be preferable if functions that do other logging end up being called, causing the metadata logging to be spread out rather than in a block.